### PR TITLE
gh-126061: add PyLong_IsPositive/Negative/Zero/GetSign functions to `refcounts.dat`

### PR DESCRIPTION
**Pull request by rruuaanng**
----
📚 Documentation preview 📚: https://cpython-previews--126788.org.readthedocs.build/

This PR completes the proposal in https://github.com/python/cpython/pull/126065, but it only adds GetSign/IsZero/IsPositive/IsNegative

Although it is a draft PR, it is open for review. For other recently added C-API, I need to spend a little time looking for them, if there are no such requirements, it can be merged directly.

* Issue: gh-126061

### CODE REVIEW

**Comment by skirpichev** [comment]
> This PR completes the proposal in https://github.com/python/cpython/pull/126065, but it only adds GetSign/IsZero/IsPositive/IsNegative

I think it's enough.

**Comment by rruuaanng** [comment]
Then I will open this PR to hear more opinions :)

cc @vstinner

File: Doc/data/refcounts.dat [hunk 1]
```diff
@@ -1284,6 +1284,19 @@ PyLong_FromUnsignedLong:unsignedlong:v::
...
+
+PyLong_IsNegative:int:::
+PyLong_IsNegative:PyObject*:obj:0:
+
+PyLong_IsZero:int:::
+PyLong_IsZero:PyObject*:obj:0:
+
+PyLong_GetSign:int:::
+PyLong_GetSign:PyObject*:v:0:
+PyLong_GetSign:int*:sign::
```
Comment (skirpichev) [inline Doc/data/refcounts.dat:1298]: Lets remove this for now.  I'm not sure about fate of this function.

**Review by skirpichev — CHANGES_REQUESTED** [review]

File: Doc/data/refcounts.dat (hunk 1 above)
Comment (rruuaanng) [reply Doc/data/refcounts.dat:1298]: It seems to have been merged, and I am a little hesitant to delete it. But if I remember correctly, it was discussed to remove it, so we can wait for victor's speech. WDYT?

Comment (vstinner) [reply Doc/data/refcounts.dat:1298]: If PyLong_GetSign() is removed, it can be removed from Doc/data/refcounts.dat as well.

Comment (rruuaanng) [reply Doc/data/refcounts.dat:1298]: In that case, I'm marking this review as resolved. We can remove `refcounts.dat` in the future when we remove GetSign.

**Review by vstinner — APPROVED** [review]

**Review by skirpichev — APPROVED** [review]

**Comment by vstinner** [comment]
Merged, thanks @rruuaanng.

**Comment by rruuaanng** [comment]
A little help. For CPython :)

**Pull request merged**